reports/pharmacy_first_data_development.Rmd

```diff
@@ -22,8 +27,15 @@ df_pf_codes_counts <- read_csv(
 df_codes_counts <- read_csv(
```

Comment:
it may be useful to change the name for the existing data to clarify the difference to the new import

```suggestion
df_codes_count_measures <- read_csv(
```